### PR TITLE
Add new IT workflow for ARM64 (TAR) and change the setup runner service script to allow multi arch TAR

### PR DESCRIPTION
**Pull request by sreekarjami**
*Issue #, if available:*

*Description of changes:*
Add new IT workflow for ARM64 (TAR) and change the setup runner service script to allow multi arch TAR. Change the async plugin to true for release candidate. 

*Test Results:*
X64 (TAR):https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/529147838
ARM64 (TAR): https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/529105596

**Note: If this PR is related to Helm, please also update the README for related documentation changes. Thanks.**
**https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/helm/README.md**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/staging-test-tar-arm64.yml [hunk 1]
```diff
@@ -0,0 +1,287 @@
+name: Process TAR Artifacts
```
Comment (gaiksaya) [inline .github/workflows/staging-test-tar-arm64.yml:1]: change to `Test ARM64 TAR artifacts`

File: .github/workflows/staging-test-tar-arm64.yml [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+on:
+  # schedule:
+  #   - cron: '0 10 * * *'
+  repository_dispatch:
+    types: [staging-build-tar]
```
Comment (gaiksaya) [inline .github/workflows/staging-test-tar-arm64.yml:7]: change to `staging-test-tar-arm64` consistent with workflow name as same as others

File: .github/workflows/staging-test-tar-arm64.yml (hunk 1 above)
Comment (sreekarjami) [reply .github/workflows/staging-test-tar-arm64.yml]: Changed.

File: .github/workflows/staging-test-tar-arm64.yml (hunk 2 above)
Comment (sreekarjami) [reply .github/workflows/staging-test-tar-arm64.yml]: Changed.

File: release-tools/scripts/setup_runners_service.sh [hunk 3]
```diff
@@ -18,7 +19,7 @@
 ###############################################################################################
 
 # This script allows users to manually assign parameters
-if [ "$#" -ne 2 ] || [ "$1" = "--help" ] || [ "$1" = "-h" ]
+if [ "$#" -lt 2 ] || [ "$1" = "--help" ] || [ "$1" = "-h" ]
 then
   echo "Please assign 2 parameters when running this script"
   echo "Example: $0 \$SETUP_DISTRO \$SETUP_ACTION"
```
Comment (peterzhuamazon) [inline release-tools/scripts/setup_runners_service.sh:25]: Please also update the help message with $ARCHITECTURE

File: .github/workflows/staging-test-tar-arm64.yml (hunk 2 above)
Comment (gaiksaya) [reply .github/workflows/staging-test-tar-arm64.yml]: Its `staging-test-tar-arm64`

**Comment by peterzhuamazon** [comment]
Where is the arm64 tarball building part? In another PR?

File: .github/workflows/staging-test-tar-arm64.yml (hunk 2 above)
Comment (sreekarjami) [reply .github/workflows/staging-test-tar-arm64.yml]: Changed it to :staging-test-tar-arm64

File: release-tools/scripts/setup_runners_service.sh (hunk 3 above)
Comment (sreekarjami) [reply release-tools/scripts/setup_runners_service.sh:25]: Added a message that the ARCHITECTURE is optional.

Comment (peterzhuamazon) [reply release-tools/scripts/setup_runners_service.sh:25]: Where is the message? not seeing it.

**Review by gaiksaya — APPROVED** [review]

File: release-tools/scripts/setup_runners_service.sh [hunk 4]
```diff
@@ -22,8 +22,8 @@
 if [ "$#" -lt 2 ] || [ "$1" = "--help" ] || [ "$1" = "-h" ]
 then
   echo "Please assign atleast 2 parameters when running this script"
-  echo "Example: $0 \$SETUP_DISTRO \$SETUP_ACTION"
-  echo "Example: $0 \"zip | docker | deb | rpm\" \"--es | --es-nosec | --kibana | --kibana-nosec \" \"ARCHITECTURE (optional)\""
+  echo "Example: $0 \$SETUP_DISTRO \$SETUP_ACTION \$ARCHITECTURE (optional)"
+  echo "Example: $0 \"zip | docker | deb | rpm\" \"--es | --es-nosec | --kibana | --kibana-nosec \" \"arm64 (optional)\""
```
Comment (sreekarjami) [inline release-tools/scripts/setup_runners_service.sh:26]: @peterzhuamazon  Are we good with this message.

File: release-tools/scripts/setup_runners_service.sh (hunk 3 above)
Comment (sreekarjami) [reply release-tools/scripts/setup_runners_service.sh]: I added a comment where the change is made. Please reply.

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**